test/integration/master_routes_test.go

```diff
@@ -63,11 +64,17 @@ var expectedIndex = []string{
 	"/apis/project.openshift.io/v1",
 	"/apis/quota.openshift.io",
 	"/apis/quota.openshift.io/v1",
+	"/apis/rbac.authorization.k8s.io",
+	"/apis/rbac.authorization.k8s.io/v1alpha1",
```

Comment:
This is broken upstream: disabled apigroup-versions get their discovery endpoint. Preparing a patch.